moving_nest/fv_moving_nest.F90

```diff
@@ -2538,13 +2678,13 @@ subroutine assign_p_grids(parent_geo, p_grid, position)
           if (p_grid(i,j,1) .ne. 0.0) num_vals = num_vals + 1
         enddo
       enddo
-      
+
       if (num_zeros .gt. 0) print '("[INFO] WDR set p_grid_v npe=",I0," num_zeros=",I0," full_zeros=",I0," num_vals=",I0" nxp=",I0," nyp=",I0," parent_geo%lats(",I0,",",I0,")"," p_grid(",I0,",",I0,",2)")', mpp_pe(), num_zeros, full_zeros, num_vals, parent_geo%nxp, parent_geo%nyp, ubound(parent_geo%lats,1), ubound(parent_geo%lats,2), ubound(p_grid,1), ubound(p_grid,2)
-      
-      
-      
+
+
+
```

Comment:
@wramstrom, in general coding style wise, I would suggest no avoid using more than 2 empty lines (unless absolutely needed) between code blocks.

There are also other locations in this set of changes with more than 2 empty lines if you would like to help to update.